app/scripts/modules/google/src/loadBalancer/configure/http/basicSettings/basicSettings.component.html

```diff
@@ -39,12 +39,27 @@
           </account-select-field>
         </div>
       </div>
-      <div class="form-group">
```

Comment:
Is this safe to remove? I see we're only conditionally rendering the `gce-region-select-field` if the load balancer `isInternal`.

Reply:
Yes, External HTTP LB is always global, so I think this form is meaningless.
Sorry for unrelated change for this PR. Should I revert it?

Reply:
Oh I see, thanks for the explanation! You don't need to revert it, but in future PRs it's helpful to separate out refactors not related to the main change into their own commits if possible!